src/App.jsx

```diff
@@ -14,70 +15,78 @@ export const goods = [
   'Garlic',
 ];
 
-export const App = () => (
-  <main className="section container">
-    <h1 className="title is-flex is-align-items-center">No goods selected</h1>
+export const Good = ({ good, selectedGood, setIsSelected }) => {
+  const addGood = () => {
+    if (selectedGood === good) {
+      setIsSelected(null);
+    } else {
+      setIsSelected(good);
+    }
+  };
 
-    <h1 className="title is-flex is-align-items-center">
-      Jam is selected
+  return (
+    <tr
+      data-cy="Good"
+      className={selectedGood === good ? 'has-background-success-light' : ''}
+    >
+      <td>
+        <button
+          data-cy={selectedGood === good ? 'RemoveButton' : 'AddButton'}
+          type="button"
+          className={`button${selectedGood === good ? ' is-info' : ''}`}
+          onClick={addGood}
+        >
+          {selectedGood === good ? '-' : '+'}
+        </button>
```

Comment:
You don't need to check if `selectedGood === good` several times. Just create 2 separate buttons and add the conditions at the beginning